sdks/java/io/cdc/src/main/java/org/apache/beam/io/cdc/DebeziumIO.java

```diff
@@ -18,9 +18,15 @@
 package org.apache.beam.io.cdc;
 
 import com.google.auto.value.AutoValue;
+
+import io.debezium.connector.sqlserver.SqlServerConnector;
```

Comment:
Why do you need to import this?

Reply:
It is not a required dependency, let me remove unused imports inn DebeziumIO class.